eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -311,9 +311,8 @@ jobs:
     displayName: Generate Code Coverage Data
     condition: and(succeeded(), ne(variables['CODE_COVERAGE'], 'disabled'), ne(variables['CODE_COVERAGE'], ''))
 
-  - task: PublishCodeCoverageResults@1
+  - task: PublishCodeCoverageResults@2
     inputs:
-      codeCoverageTool: Cobertura
       summaryFileLocation: '$(Build.SourcesDirectory)/**/Cobertura.xml'
```

Comment:
Is Covertura the default tool now?

Reply:
Yes I think it auto-detects the results.